pipelines/kubeflow_pipelines_component_examples/run_notebook_component/run_notebook_using_papermill.pipeline

```diff
@@ -9,23 +9,77 @@
+              "x_pos": 16.99999237060547,
+              "y_pos": 141.5
+            }
+          },
+          "inputs": [
+            {
+              "id": "inPort",
+              "app_data": {
+                "ui_data": {
+                  "cardinality": {
+                    "min": 0,
+                    "max": -1
+                  },
+                  "label": "Input Port"
+                }
+              }
+            }
+          ],
+          "outputs": [
+            {
+              "id": "outPort",
+              "app_data": {
+                "ui_data": {
+                  "cardinality": {
+                    "min": 0,
+                    "max": -1
+                  },
+                  "label": "Output Port"
+                }
+              }
+            }
+          ]
+        },
+        {
+          "id": "d3ecde8b-8838-402f-9a45-764a0539e273",
+          "type": "execution_node",
+          "op": "run_notebook_using_papermill_Runnotebookusingpapermill",
+          "app_data": {
+            "component_parameters": {
+              "notebook": {
+                "node_id": "5093aaa3-7ce0-4654-93c8-ea67ebe3fb83",
+                "output_key": "elyra_output_data",
+                "label": "Download data: (Output) Data"
+              },
               "parameters": "{'multiplier':2, 'data_url': 'https://raw.githubusercontent.com/elyra-ai/examples/master/pipelines/kubeflow_pipelines_component_examples/run_notebook_component/data/data-1.csv'}",
               "packages_to_install": "['pandas']",
-              "input_data": "some string data"
+              "input_data": "some string data",
+              "output_notebook": "",
+              "output_output_data": ""
             },
             "label": "",
-            "component_source": "kfp/run_notebook_using_papermill.yaml",
+            "component_source": "/Users/akchin/opt/anaconda3/envs/test-package/share/jupyter/components/kfp/run_notebook_using_papermill.yaml",
```

Comment:
This will not work as is because of the proprietary path reference

Reply:
Thx for the catch, had duplicate component listing in my registry and musta dragged the wrong one. Updated with source pointing to component available in KFP github.